### PR TITLE
HPCC-15083 Use the c++11 enum syntax in hqlexpr.hpp

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@richardkchapman please review

File: ecl/hql/hqlexpr.hpp [hunk 1]
```diff
@@ -803,12 +803,6 @@ enum ExprPropKind
 };
 
 
-#ifdef ENABLE_ENUM_TYPE
```
Comment (richardkchapman) [inline ecl/hql/hqlexpr.hpp]: Should also remove the #define at the top of this file I think

**Comment by richardkchapman** [comment]
@ghalliday Back to you

**Comment by ghalliday** [comment]
@richardkchapman removed #define and squashed.

**Pull request merged**